core/tests/test_utils.py

```diff
@@ -197,11 +240,20 @@ def _restore_stashed_user_env(self):
         self.stashed_user_env = None  # pylint: disable=attribute-defined-outside-init
 
     def login(self, email, is_super_admin=False):
+        """Assign environment variables for login credentials of user.
+
+        Args:
+            email: string. user email address. Also used to retrieve user id.
```

Comment:
email: str. The email address of the user.

(That's sufficient; no need to go into implementation details. The caller should be able to read this docstring and then figure out how to call the method.)

Reply:
done

Reply:
done